.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vcpkg"]
+	path = vcpkg
+	url = https://github.com/microsoft/vcpkg.git
```

Comment:
let's add
`fetchRecurseSubmodules = false` here

Since someone just doing
`git clone microsoft/stl` probably doesn't want this submodule

Reply:
done